### ISSUE TITLE
FireFox+Nextcloud 15 Sync shows a 404 not found error since upgrade to fx 65.0/66.0

### ISSUE THREAD

**Issue by rcmcronny** [subdavis/Tusk#294]
### Steps to reproduce - current behaviour

Type in the unlock passwort, a field above with the message 

**" Invalid response: 404 Not Found"** 

is shown and the database is NOT unlocked.

### What is the expected behaviour

Type in the unlock password, the matching elements should be shown

#### Additional remarks

**Browser Console shows:**

_Invalid response: 404 Not Found popup.build.js:8:268736
     unlock moz-extension://d57aaab5-e3fd-49a4-960b-20a976f36585/build/popup.build.js:8 

Sicherheitsfehler: Inhalt auf moz-nullprincipal:{9855ef58-6578-44fd-8ae0-96f252ad4ff4} darf keine Daten von moz-extension://d57aaab5-e3fd-49a4-960b-20a976f36585/build/popup.build.js laden.
NS_ERROR_DOM_BAD_URI: Component returned failure code: 0x805303f4 [nsIWebNavigation.loadURI] viewSource-content.js:284_

#### Environment 
  Operating System: Linux Gentoo Desktop
  Browser: Firefox 65.0about:version)
  Storage: Nextcloud 15

Before the upgrade to firefox 65.0 this worked, so it ha to do with the upgrade i suspekt.

**Comment by rcmcronny** [comment]
the same issue is with firefox-66 now, any fix or hint ?

**Comment by henryptung** [comment]
@rcmcronny Are you using the shared link provider? If so, you may want to take a look at https://github.com/subdavis/Tusk/issues/307 and https://github.com/subdavis/Tusk/pull/308.

**Comment by rcmcronny** [comment]
nope, i use the WebDAV provider with the suggested secured meassures.

**Comment by fengelhardt** [comment]
I first got the same 404 message with my WebDAV database. It have removed my database and tried re-adding it, but I now get an error in the "Manage Databases" tab:

`Error: Cannot request origin permission for https://<my WebDAV URI> since it was not declared in the manifest`

Maybe some security policies have changed in Firefox? Or maybe a Nextcloud issue?

**Comment by subdavis** [comment]
This has been fixed by #357, but not yet pushed to the Firefox addon store.
